src/ui/hooks/usePlayerEvent.tsx

```diff
@@ -0,0 +1,16 @@
+import { DependencyList, useEffect } from "react";
+import type { PlayerEventMap, THEOplayer } from "react-native-theoplayer";
+
+export function usePlayerEvent<K extends keyof PlayerEventMap>(
```

Comment:
I don't think this is a good abstraction.

Delete this, and use our existing `useCurrentTime` hook instead.